### PR TITLE
Feature/emails

### PR DESCRIPTION
**Pull request by Elliott127**
EMAILS WORK! (insert blood on the leaves by kanye here)

### CODE REVIEW

File: APP/MauiProgram.cs [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ﻿using APP.States;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
 using APP.Services;
 using APP.Interfaces;
-using APP.Data;
+using APP.Services.Email;
```
Comment (Douglas-Thornton) [inline APP/MauiProgram.cs:6]: Email Service uses the namespace app.services so this can be removed.

File: APP/Pages/Meetup.razor [hunk 2]
```diff
@@ -1,39 +1,42 @@
 ﻿@using APP.Data.Models;
 @using APP.Interfaces;
+@using Microsoft.Extensions.Configuration;
 @using APP.Services;
+@inject IConfiguration Configuration;
 @inject LoggedUserState loggedUserState
```
Comment (Douglas-Thornton) [inline APP/Pages/Meetup.razor:7]: Requires the
@using APP.States;
@using System.ComponentModel;
namespaces to access the loggeduserState and propertyChanged events.

**Review by Douglas-Thornton — CHANGES_REQUESTED** [review]
Left a review

File: APP/Pages/Meetup.razor (hunk 2 above)
Comment (Elliott127) [reply APP/Pages/Meetup.razor:7]: added

**Review by Douglas-Thornton — APPROVED** [review]

**Pull request merged**